tests/ops/test_channel_ops.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the available built-in quantum channels.
+"""
+import pytest
+import functools
+import numpy as np
+from numpy.linalg import multi_dot
+
+import pennylane as qml
+from pennylane.ops import channel
+
+ch_list = [channel.AmplitudeDamping,
+           channel.GeneralizedAmplitudeDamping,
+           channel.PhaseDamping,
+           channel.DepolarizingChannel,
+           ]
+
+X = np.array([[0, 1], [1, 0]])
+
+class TestChannels:
+    """Tests for the quantum channels"""
+
+    @pytest.mark.parametrize("ops", ch_list)
```

Comment:
> I think the tests aren't working due to line 887 in `operation.py`:
> `if any(p > 1 for p in params):`, since it doesn't like comparing `p` being complex to 1 (an int). Maybe we could have another check:
> `if any(np.iscomplex(p) for p in params): ...`

I do not think `p` can be complex. it can be real, natural or an array. It might be int and float issue. i will try with 1.0 instead :)